app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    sepp_command = command.split()
+
+    source = sepp_command[1]
+    destination = sepp_command[2]
+
+    path_to_move = destination.split("/")
+
+    new_dir = ""
```

Comment:
The variable name `new_dir` could be more descriptive. Consider renaming it to something like `current_path` or `directory_path` to better convey its purpose.